diff --git a/src/reflect/value.go b/src/reflect/value.go
index cb784df6..06714487 100644
--- a/src/reflect/value.go
+++ b/src/reflect/value.go
@@ -751,8 +751,26 @@ func MakeSlice(typ Type, len, cap int) Value {
 	panic("unimplemented: reflect.MakeSlice()")
 }
 
+// Zero returns a Value representing the zero value for the specified type.
+// The result is different from the zero value of the Value struct,
+// which represents no value at all.
+// For example, Zero(TypeOf(42)) returns a Value with Kind Int and value 0.
 func Zero(typ Type) Value {
-	panic("unimplemented: reflect.Zero()")
+	if typ == nil {
+		panic("reflect: Zero(nil)")
+	}
+	t := typ.(rawType)
+
+	switch t.Kind() {
+	case Func, Chan, Map, Ptr, Slice, UnsafePointer, Interface:
+		return Value{t, nil, valueFlagExported}
+	case Bool, Int, Int8, Int16, Int32,Int64, Uint,Uint8,Uint16,Uint32,Uint64,Float32,
+		Float64, Complex64, Complex128, String, Array, Struct:
+		val := alloc(typ.Size(), nil)
+		return Value{t, val, valueFlagExported&valueFlagIndirect}
+	default: // not implemented: Func
+		panic(&ValueError{Method: "Pointer"})
+	}
 }

 // New is the reflect equivalent of the new(T) keyword, returning a pointer to a
diff --git a/targets/wasi.json b/targets/wasi.json
index dfec729e..fe953486 100644
--- a/targets/wasi.json
+++ b/targets/wasi.json
@@ -1,7 +1,7 @@
 {
 	"llvm-target":   "wasm32-unknown-wasi",
 	"cpu":           "generic",
-	"features":      "+bulk-memory,+nontrapping-fptoint,+sign-ext",
+	"features":      "+bulk-memory",
 	"build-tags":    ["tinygo.wasm", "wasi", "runtime_memhash_leveldb"],
 	"goos":          "linux",
 	"goarch":        "arm",
@@ -10,9 +10,7 @@
 	"scheduler":     "asyncify",
 	"default-stack-size": 16384,
 	"cflags": [
-		"-mbulk-memory",
-		"-mnontrapping-fptoint",
-		"-msign-ext"
+		"-mbulk-memory"
 	],
 	"ldflags": [
 		"--allow-undefined",
